src/main.rs

```diff
@@ -80,26 +84,46 @@ async fn main() -> anyhow::Result<()> {
 
                             fs::copy(file, &new_path).with_context(|| {
                                 format!(
-                                    "failed to copy {} to {}",
+                                    "Failed to copy {} to {}",
                                     file.display(),
                                     new_path.display()
                                 )
                             })?;
 
                             app_info.xdc_blob_dir = new_path;
```

Comment:
why is this called "dir" here btw?  Isn't it the path to the app `.xdc` file?

Reply:
yes, should be path

Reply:
But that's also a refactoring I would like to do outside this pr